### DISCUSSION TITLE
ap itu flmt

### DISCUSSION THREAD

**Discussion by portRafi** [portRafi/KasirCerdas#35] [Announcements]
apa itu filament

**Comment by miiiwdy** [comment]
Filament PHP adalah framework atau toolset berbasis PHP yang digunakan untuk membangun aplikasi admin panel atau dashboard dengan cepat dan efisien. Filament sangat terintegrasi dengan Laravel, sehingga sering digunakan oleh para pengembang yang ingin membuat panel administrasi yang cepat dan mudah untuk dikelola dalam proyek Laravel mereka.
